File: ocaml/quine.ml
(* Quine in OCaml
https://code.golf/quine#ocaml
Copyright (c) Kira K. 2023
BEGIN*)
(fun s -> Printf.printf "%s%S;;" s s) "(fun s -> Printf.printf \"%s%S;;\" s s) ";;
(*END*)

File: rust/src/quine.rs
/* Quine in Rust
https://code.golf/quine#rust
Copyright (c) Kira K. 2023
*/
//BEGIN
fn main(){let x="fn main(){let x=";let y="print!(\"{}{:?};let y={:?};{}\",x,x,y,y)}";print!("{}{:?};let y={:?};{}",x,x,y,y)}
//END
